topics/chrome/index.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: google-chrome
+related: firefox,safari,edge,opera,chromium,browser,chrome-extension
+created_by: Google
+display_name: Chrome
+github_url: https://github.com/googlechrome
+logo: chrome.png
+released: September 2, 2008
+short_description: Chrome is a web browser from the tech company Google.
+topic: chrome
+url: https://www.google.com/chrome/
+---
+
+Chrome is the most popular web browser worldwide as of mid-2017, made by the tech company Google. It's available for most operating systems including Windows, macOS and Linux and on multiple platforms such as the desktop, phones and tablets.
+
+Chrome boasts a minimalistic UI and was the first browser to feature "tabs" above the address bar, a convention that was later implemented in other browsers. Other popular features include things such as Ingonito mode, Tab sandboxing and a Web Store to install extenions and themes from.
+
+Although Chrome is not open source, the majority of the source code is avaible under the Chromium moniker 
```

Comment:
Typo: `avaible`

Can you add a period after "moniker"?

Reply:
Damn! That's one of those english words I never seem to get right 😄